router/local_peer.go

```diff
@@ -27,54 +27,54 @@ func NewLocalPeer(name PeerName, nickName string, router *Router) *LocalPeer {
 	return peer
 }
 
-func (peer *LocalPeer) Forward(dstPeer *Peer, frame []byte, dec *EthernetDecoder) error {
-	return peer.Relay(peer.Peer, dstPeer, frame, dec)
+func (peer *LocalPeer) Forward(dstPeer *Peer, key PacketKey) FlowOp {
```

Comment:
It feels a bit odd to me to have these four methods (Forward/Broadcast/Relay/RelayBroadcast)  in local_peer.go rather than router.go - thoughts?